pyproject.toml

```diff
@@ -1,8 +1,55 @@
-[build-system]
-requires = [
-    "setuptools>=40.9.0",
-    "wheel",
+[project]
+name = "massedit"
+description = "Edit multiple files using Python text processing modules"
+# UPDATE massedit.py
+version = "0.69.1"
+authors = [{name = "elmotec", email = "elmotec@gmx.com"}]
+license = {text = "MIT"}
+keywords = [
+     "sed",
+     "editor",
+     "stream",
+     "python",
+     "edit",
+     "mass"
 ]
-build-backend = "setuptools.build_meta"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: MIT License",
+    "Environment :: Console",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 2.7",
+    "Programming Language :: Python :: 3.4",
```

Comment:
Also remove line 21 and 22 here.  Those versions are no longer supported.  I'll put a separate PR to add 3.10, 3.11. Man, time flies.

Reply:
And in [.github/workflows/python-app.py](https://github.com/elmotec/massedit/blob/0229b40361ed6e752beb81a69f9f78d92c4906d5/.github/workflows/python-app.yml#L18) as well.